src/universal/modules/meeting/components/MeetingCheckIn/MeetingCheckIn.js

```diff
@@ -55,11 +67,15 @@ const MeetingCheckin = (props) => {
               nextMemberName={nextMemberName}
             /> :
             <div className={css(styles.hint)}>
-              <MeetingFacilitationHint>
+              <MeetingFacilitationHint showEllipsis={!nextMemberName || !isMyMeetingSection}>
                 {nextMemberName ?
-                  <span>{'Waiting for'} <b>{currentMember.preferredName}</b> {'to share with the team'}</span> :
-                  <span>{'Waiting for'}
-                    <b>{facilitatorName}</b> {`to advance to ${actionMeeting.updates.name}`}</span>
+                  <span>
+                    {isMyMeetingSection ?
```

Comment:
This logic is kinda funny. So it tells you to share, unless you're the last person?
i know this isn't new, but it still seems odd, especially because the last people are new people, so the people who need direction the most are the ones being told "waiting for facilitator to advance"
Maybe we change the wording after 30 seconds?

Reply:
I propose the following:
https://docs.google.com/spreadsheets/d/1U4IwUGMokwaIOjC0WdWgtp2Qizf6dL4ueHkAHTKo5M8/edit#gid=0

Reply:
Yeah, I’m up for improving this! I see what you mean.
- Teammate: “So and so is sharing” or “Waiting for facilitator to advance after so and so shares”
- You: “Your turn to share”
- Facilitator: “Advance once so and so gives their check-in”
- You are facilitator: “Give your check-in then advance”

I really don’t know, just brainstorming. And a timing function may be worth a try.